Maman11/Q2/my_suffix.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#define STRING_LENGTH 127
+
+int suffix(char str[], char c){
+    bool read = false;
+    int suffix = 0, len = strlen(str), i = 0;
+    while (str[i]){
+        if (c == str[i]){
+            read = true;
+        }
+        if (read){
+            print_ending(&str[i]);
+            read = false;
+            suffix += 1;
+
+        }
+
+        i++;
+    }
+    return suffix;
```

Comment:
suffix is not a good variable name for this. 
This is a counter for the number of suffixes that the string has